app/assets/stylesheets/pages/_informations.scss

```diff
@@ -0,0 +1,48 @@
+.infos-wrapper {
+  max-width: 800px;
+  margin: auto;
+  max-height: 300px;
+  margin-bottom: 48px;
+}
```

Comment:
Can you rename this `wrapper-800` and move it into a `wrapper.scss` file into components folder 🙏 

I think it will be reuse on contact page, so let's anticipate a bit and store it at the good place.

Reply:
Also I think you don't need the `max-height ` property.